stylelint-polaris/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "@shopify/stylelint-polaris",
+  "description": "Polaris Design System Stylelint tooling",
+  "version": "0.0.0",
+  "private": false,
+  "license": "SEE LICENSE IN LICENSE.md",
+  "author": "Shopify <dev@shopify.com>",
+  "repository": "https://github.com/Shopify/polaris",
+  "bugs": {
+    "url": "https://github.com/Shopify/polaris/issues"
+  },
+  "publishConfig": {
+    "access": "public",
+    "@shopify:registry": "https://registry.npmjs.org"
+  },
+  "keywords": [
+    "shopify",
+    "polaris",
+    "stylelint"
+  ],
+  "main": "index.js",
+  "engines": {
+    "node": ">=16.13.0"
+  },
+  "scripts": {
+    "gen-polaris-vars": "node ./scripts/gen-polaris-vars.js",
+    "postinstall": "yarn gen-polaris-vars",
+    "build": "echo '@shopify/stylelint-polaris does not have a build script.'",
+    "type-check": "tsc --noEmit -p tsconfig.json",
+    "prepublishOnly": "yarn gen-polaris-vars && yarn type-check"
```

Comment:
Looking forward to getting rid of this when we have a pan-repo type-check command :)